package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "theia",
+  "name": "theia-ide",
   "version": "0.0.1-alpha.1",
   "description": "Theia - JavaScript IDE Framework",
```

Comment:
+1. How about
`Theia is a cloud & desktop IDE framework implemented in TypeScript.`?